jakarta-ee-support/index.html.haml

```diff
@@ -0,0 +1,14 @@
+---
+layout: rightcol
+rightcol_partial: rightcol-documentation.html.haml
+title:  Jakarta EE Support
+---
+%h1
+  Jakarta EE Support
+%p
+  Narayana implements the Jakarta Transactions 2.0 specification. 
+%p
+  Narayana is a 100% Java implementation of a distributed transaction management system based on the Sun Microsystems Java Transaction Service (JTS) standard. Our implementation of the JTS utilizes the Object Management Group's (OMG) Object Transaction Service (OTS) model for transaction interoperability as referenced in Jakarta Enterprise Beans 8.1.3 and leads the market in providing many advanced features such as fully distributed transactions and ORB portability with POA support.
+%p
+  Narayana can be run both as an embedded distributed service of an application server (e.g., WildFly), affording the user all the added benefits of the application server environment such as real-time load balancing, unlimited linear scalability and unmatched fault tolerance that allows you to deliver an always-on solution to your customers. It is also available as a free-standing Java Transaction Service.
```

Comment:
Doesn't embedded normally mean "embedded with the application" whereas WildFly provides Transaction services to applications.

What does "unlimited linear scalability" mean, are we able to reference any material or tests that would back up the claim. Similarly for "unmatched fault tolerance", have we analysed the competitors to see what they provide.

Isn't the "always-on solution" restricted to when it runs on OpenShift.

Reply:
A lot of the information you are asking about I expect would be in the original version (that would be deleted in this pull request): https://github.com/jbosstm/narayana.io/blob/develop/java-ee-support/index.html.haml but I did change from `JBoss` to `WildFly` here.

Reply:
Fair enough, I thought it was new text that you'd added, so in that case I approve (and trust that the statements are still accurate).